Test/vstte2012/BreadthFirstSearch.dfy

```diff
@@ -86,9 +86,10 @@ class BreadthFirstSearch<Vertex(==)>
     {
       // remove a vertex "v" from "C"
       var v :| v in C;
+      assert v != dest && C == {v} ==>
+        (Processed + {v}) + (N + (set w | w in Succ(v) && w !in V)) == R(source, d+1, AllVertices) by { reveal R(); }
       C, Processed := C - {v}, Processed + {v};
       ghost var pathToV := Find(source, v, paths);
-
```

Comment:
Is there a reason to remove these blank lines?

Reply:
No indeed that does not improve verifiability :-)